spec/acceptance/README.md

```diff
@@ -47,6 +47,20 @@ Run the test suite:
 POSTGRES_RPORT=9000 SPEC_OPTS='--tag acceptance' SPEC_HELPER_LOAD_METASPLOIT=false bundle exec rspec ./spec/acceptance/postgres_spec.rb
 ```
 
+### MySQL
+
+Run a target:
+
+```
+docker run -it --rm --publish 127.0.0.1:9000:3306 -e MYSQL_ROOT_PASSWORD=password mariadb:11.2.2
```

Comment:
Should probably pick unique ports to allow folk to run through all the tests locally without conflicting with eachother

Reply:
Ah, good shout. I'll update to `9001`.